### ISSUE TITLE
Feature request: Add MR priority support 

### ISSUE THREAD

**Issue by left4taco** [smarkets/marge-bot#153]
Suppose I have 5 MRs assigned to marge-bot, it'll by default pick the

**Comment by benjamb** [comment]
It could be an idea to allow the configuration of arbitrary labels, perhaps with the priority being set based on the order in which they are specified?

**Comment by JaimeLennox** [comment]
I would agree with @benjamb. I also think it should be optional, as there may be situations where we don't want this behaviour, as it could be easily abused by someone to always take priority over others.

**Comment by vi** [comment]
Priority merge requests also tend to be excluded from batches.
Does marge-bot support individually suppressing batching for a specific merge request?
